webknossos/tests/dataset/test_downsampling.py

```diff
@@ -260,27 +256,75 @@ def test_anisotropic_mag_calculation() -> None:
         ),
         (None, (2, 2, 1), (8, 8, 4), [(2, 2, 1), (4, 4, 2), (8, 8, 4)]),
         (None, (2, 2, 1), (8, 8, 8), [(2, 2, 1), (4, 4, 2), (8, 8, 4)]),
-    ]
-
-    for i in range(len(mag_tests)):
-        voxel_size, from_max_name, max_mag_name, scheme = mag_tests[i]
-        sampling_scheme = [Mag(m) for m in scheme]
-        from_mag = Mag(from_max_name)
-        max_mag = Mag(max_mag_name)
-
-        assert sampling_scheme[1:] == calculate_mags_to_downsample(
-            from_mag, max_mag, None, voxel_size
-        ), f"The calculated downsampling scheme of the {i+1}-th test case is wrong."
-
-    for i in range(len(mag_tests)):
-        voxel_size, finest_mag_name, from_mag_name, scheme = mag_tests[i]
-        sampling_scheme = [Mag(m) for m in scheme]
-        from_mag = Mag(from_mag_name)
-        finest_mag = Mag(finest_mag_name)
-
-        assert list(reversed(sampling_scheme[:-1])) == calculate_mags_to_upsample(
-            from_mag, finest_mag, None, voxel_size
-        ), f"The calculated upsampling scheme of the {i+1}-th test case is wrong."
+    ],
+)
+def test_mag_calculation(
+    voxel_size: Optional[Tuple[float, float, float]],
+    from_mag_name: Tuple[float, float, float],
+    max_mag_name: Tuple[float, float, float],
```

Comment:
why is there a `_name` at the end of the parameter names? For me it sounds like a `mag_name` would be something like `"mag_xyz"` and not a tuple of values to represent the mag? Is this a naming convention in wklibs?

Reply:
I don't think there is a naming convention like this in wklibs. I changed the naming of the parameters and adapted the type specification.